src/main/resources/application.conf

```diff
@@ -21,6 +21,8 @@ akka {
   }
 }
 
+akka.http.server.request-timeout = 90s
```

Comment:
I already added this since you told me about it in the config. It's nested under akka / http / server. It's at 60s, but feel free to make it 90s, just want to be consistent.

Reply:
Will remove